admin2: Added content for "Users" sub-tab in the "Rights" tab

FileEX opened the pull request:
This PR add content for "Users" sub tab in the "Rights" tab.

![image](https://github.com/multitheftauto/mtasa-resources/assets/26044224/d5e1099a-9fb0-40a3-9963-5c7b845d33d7)

jlillis reviewed (changes requested):
Looks good overall, but two minor issues:
1 - PR fails lint checks
2 - Maybe add a confirmation box (messageBox) when removing a user?

Feel free to put the rest of your work on this ACL tab in this PR if you want.

FileEX commented:
> Looks good overall, but two minor issues: 1 - PR fails lint checks 2 - Maybe add a confirmation box (messageBox) when removing a user?

Fixed issues and update texts and added a confirmation window

> Feel free to put the rest of your work on this ACL tab in this PR if you want.

I asked a few people on discord and they all answered me unequivocally that I should create a separate PR for each tab. I'd rather be safe and send the bookmarks as a separate PR than make all the bookmarks as one PR and then edit and split it all up because someone else won't like it. I will be working on more sub-tabs, but will post them in separate PRs
